mods/common/languages/en.ftl

```diff
@@ -546,7 +546,7 @@ players-label =
 
 bots-label =
     { $bots ->
-        [0] No Bots
+        [0] {""}
```

Comment:
This goes against the convention. This is not related to localisation so the logic should be handled elsewhere. 
Preferably here https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Mods.Common/Widgets/Logic/ServerListLogic.cs#L418-L423